config/routes.rb

```diff
@@ -12,6 +12,7 @@
   resources :visits, only: [:index, :show] do
     post :assignment, to: 'visits#assign'
     put :remotion, to: 'visits#remove_assignment'
+    put :complete, to: 'visits#complete'
```

Comment:
para dejarlo acorde a los otros endpoints, yo lo dejaría como `put :completion, to: `visits#complete`